core/serialization/deserializer/montage-reviver.js

```diff
@@ -1023,8 +990,9 @@ var MontageReviver = exports.MontageReviver = Montage.specialize(/** @lends Mont
                     typeof reviver[methodName] === "function" &&
                         methodName.substr(0, 5) === "revive"
                 ) {
-                    if (typeof customObjectRevivers[methodName] === "undefined") {
-                        customObjectRevivers[methodName] = reviver[methodName].bind(reviver);
+                    customObjectReviver = this.customObjectRevivers.get(methodName);
+                    if (typeof customObjectReviver === "undefined") {
+                        customObjectReviver = reviver[methodName].bind(reviver);
```

Comment:
.bind() create a new function, could we find a way to avoid doing so? That could boost perfs

Reply:
Hmm...
`customObjectReviver` is a local variable and isn't used again after it is assigned to `reviver[methodName].bind(reviver)`, so isn't that assignment actually pointless?

Reply:
Actually, I think this bind has to stay, because we're grabbing a method from a user-supplied object.
Anyway, custom object revivers aren't actually used anywhere in montage or filament/palette, so we won't see a performance boost from removing bind.